br/tests/br_views_and_sequences/run.sh

```diff
@@ -39,11 +50,20 @@ run_br restore db --db $DB -s "local://$TEST_DIR/$DB" --pd $PD_ADDR
 
 set -x
 
-views_count=$(run_sql "select count(*) c, sum(m) s from $DB.view_3;" | tail -2 | paste -sd ';')
+views_count=$(run_sql "select count(*) c, sum(m) s from $DB.view_3;" | tail -2 | paste -sd ';' -)
 [ "$views_count" = 'c: 8;s: 181' ]
 
 run_sql "insert into $DB.table_2 (c) values (33);"
 seq_val=$(run_sql "select a >= 8 and b >= 4 as g from $DB.table_2 where c = 33;" | tail -1)
 [ "$seq_val" = 'g: 1' ]
 
-run_sql "drop schema $DB"
+run_sql "insert into $DB.auto_inc values ();"
+last_id_after_restore=$(run_sql "select n from $DB.auto_inc order by n desc limit 1;" | trim_sql_result)
+[ $last_id_after_restore -gt $last_id ]
+rnd_last_id_after_restore=$(run_sql "insert into $DB.auto_rnd values ();select last_insert_id();" | trim_sql_result )
+[ $(( rnd_last_id_after_restore & 0xff )) -gt $(( last_rnd_id & 0xff )) ]
```

Comment:
```suggestion
rnd_last_id_after_restore=$(run_sql "insert into $DB.auto_rnd values ();select last_insert_id() & 0xffffffffffffff;" | trim_sql_result )
[ $rnd_last_id_after_restore -gt $last_rnd_id ]
```

`auto_random(8)` means the upper 8 bits are randomized, not that the lower 8 bits are used as counter.